cumulus/runtime/Cargo.toml

```diff
@@ -7,6 +7,7 @@ license = "Unlicense"
 homepage = "https://substrate.io"
 repository = "https://github.com/paritytech/cumulus/"
 edition = "2021"
+links = "cirrus-runtime"
```

Comment:
I just wonder why it doesn't work without `links`, `DEP_FOO_KEY=value` won't be printed if no `links` is presented when used as a dependency? I have some difficulty understanding the cargo build process ...

Reply:
Not entirely sure to be honest. It is defined in the section of the documentation with `links` and indeed environment variable doesn't propagate to dependent crate unless `links` is present. Looks like designed for more narrow use case of linking to system crates, but IMO there are more use cases like this one.